### ISSUE TITLE
Enable Airbnb jsx-a11y rules

### ISSUE THREAD

**Issue by mvolk** [mvolk/ciderref#20]
New linting rules (Airbnb style) include jsx-a11y rules that are violated by poor practices in the code base. Fix the underlying issues and enable the rules.
